misaligned.c

```diff
@@ -2,20 +2,20 @@
 #include <assert.h>
 
 int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
-        }
-    }
-    return i * j;
+	const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+	const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+	int i = 0, j = 0;
+	for(i = 0; i < 5; i++) {
+		for(j = 0; j < 5; j++) {
+			printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);

```

Comment:
Separate the forming (the `i * 5 + j` part) into a separate function and write an assert for it in `main`